Sources/MessagesViewController.swift

```diff
@@ -75,6 +75,13 @@ open class MessagesViewController: UIViewController {
     open override func viewDidAppear(_ animated: Bool) {
         super.viewDidAppear(animated)
         addKeyboardObservers()
+        messageInputBarCopy?.removeFromSuperview()
+        messageInputBarCopy = nil
```

Comment:
Can you put these two things in its own private method `removeMessageInputBarCopy()`?